cv_next/components/layout/navbar.tsx

```diff
@@ -15,98 +18,111 @@ const navLinks = [
   }
 ]
 
-
 export default function Navbar() {
-  const [navbar, setNavbar] = useState(false)
+  const [isNavbarMenuOpen, setIsNavbarMenuOpen] = useState(false)
+  const [isSettignsOpen, setIsSettignsOpen] = useState(false)
+
 
   const handleClick = async () => {
-    setNavbar(false)
+    setIsNavbarMenuOpen(false)
   }
 
   useEffect(() => {
-    if (navbar) {
+    if (isSettignsOpen) {
       document.body.style.overflow = "hidden"
```

Comment:
Ok why are we accessing DOM directly here? Is there a real reason? @Aradon267 please look into this

Reply:
Don't know the difference that it makes(tried removing it locally)... @PeroG28 any idea If this is necessary?

Reply:
Don't really know the use of this
As I said, I've deleted it locally and nothing broke or change so deleting these lines